cli/src/main/java/com/devonfw/tools/ide/tool/IdeasyCommandlet.java

```diff
@@ -308,7 +306,7 @@ protected void configureWindowsTerminalGitBash() {
     }
 
     try {
-      String bashPath = this.context.findBash();
+      String bashPath = this.context.findBashRequired();
```

Comment:
The method `findBashRequired()` will throw an exception if bash was not found.
This step is a convenience feature that is rather nice-to-have and should not stop the installation process with an error.
Also the following `if` then becomes pointless.
```suggestion
      String bashPath = this.context.findBash();
```